src/components/FindWallet/WalletTable/index.tsx

```diff
@@ -310,6 +310,8 @@ const WalletTable = ({
           )
 
           const showMoreInfo = (wallet) => {
+            // Prevent expanding the wallet more info section when clicking on the "Visit website" button
+
```

Comment:
Last thing, just going to move this comment next to the onClick handler where it's referencing

```suggestion
```